javascript/javascript1/week1/Ahousepriceestimator.js

```diff
@@ -0,0 +1,37 @@
+// Project 3
+//A house price estimator
+let houseprice;
+let volumeinMeters;
+
+// 1-peter's house price calculation
+let gardenSizeInM2 = 100;
+let width = 8;
+let depth = 10;
+let height = 10;
+let actualhouseprice = 2500000;
```

Comment:
Variable names are conventionally `camelCased` to make them more readable, especially if they're composed of multiple words. How will this variable look like if this convention is followed?